contracts/staking/GoodFundManager.sol

```diff
@@ -209,13 +209,14 @@ contract GoodFundManager is DAOContract {
 	function collectInterest(address[] memory _stakingContracts) public {
 		uint256 initialGas = gasleft();
 		_reserveHasInitialized();
-		ERC20 iToken = ERC20(nameService.getAddress("CDAI"));
+		cERC20 iToken = cERC20(nameService.getAddress("CDAI"));
+		ERC20 daiToken = ERC20(nameService.getAddress("DAI"));
 		address reserveAddress = nameService.addresses(nameService.RESERVE());
-		// iToken balance of the reserve contract
-		uint256 currentBalance = iToken.balanceOf(reserveAddress);
+		// DAI balance of the reserve contract
+		uint256 currentBalance = daiToken.balanceOf(reserveAddress);
```

Comment:
this means that all staking contracts must contribute in DAI
otherwise you need to also calculate starting cDAI balance

Reply:
Yes interests collected in DAI

Reply:
also when iToken is already cdai?

Reply:
Same procedure

Reply:
les allow also cDAI, see newer comments about it

Reply:
Fixed like that